grabcut_nodes.py

```
@@ -709,69 +791,92 @@ def refine_mask(self, image: torch.Tensor, mask: torch.Tensor,
+                        alpha = 1.0 - alpha
+                        rgba[:, :, 3] = (alpha * 255).astype(np.uint8)
+                    for c in range(3):
+                        rgb[:, :, c] *= alpha
+                    rgb_tensor = torch.from_numpy(rgb).to(dtype=torch.float32)
+                    alpha_tensor = torch.from_numpy(alpha).to(dtype=torch.float32)
+                    refined_images.append(rgb_tensor)
+                    refined_masks.append(alpha_tensor)
+                else:
+                    refined_images.append(image[i])
+                    refined_masks.append(mask[i])
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)